test/org/sagebionetworks/bridge/play/controllers/ConsentControllerMockedTest.java

```diff
@@ -163,6 +166,7 @@ public void canWithdrawConsent() throws Exception {
         JsonNode node = BridgeObjectMapper.get().readTree(response);
         assertEquals("User has been withdrawn from the study.", node.get("message").asText());
         
+        // Should call the service and withdraw
         verify(consentService).withdrawConsent(any(Study.class), any(User.class), captor.capture(), any(Long.class));
         assertEquals("Because, reasons.", captor.getValue().getReason());
     }
```

Comment:
Clarification: I think we should still test that we call cacheProvider.setUserSession(), otherwise we have no automated way to test that.